src/cocotb/regression.py

```diff
@@ -861,8 +941,8 @@ def generate_tests(self, prefix="", postfix=""):
                 else:
                     doc += "\t{}: {}\n".format(optname, repr(optvalue))
 
-            self.log.debug(
-                'Adding generated test "%s" to module "%s"' % (name, mod.__name__)
+            self.log.info(
```

Comment:
Changing the log level is a bit of an unrelated behavioral change which I don't mind but wanted to point out.